jira_scraper/core/scraper.py

```diff
@@ -158,6 +171,7 @@ def run(self, backup_path: str = "jira_all_bugs.pickle"):
 
         df = self.process_issues_to_dataframe(issues)
         print(df.info())
+        self._dbg_print(df)
```

Comment:
Why do we want  to print all dataframes one by one by default? I think we can add a debug flag to the command if someone wants to print everything. I don't see anyone running the tool after setting PYTHONOPTIMIZE env var.

Reply:
Agreed here.

Reply:
Printing the summary, `df.info()`, is imho enough. It gives us all the things we need.

Reply:
__debug__ is true only if the code is not optimized, so it will not run by default. Anyways, removed the call to the method. Thanks for the review :-)